javascript/javascript1/week1/ez.js

```diff
@@ -0,0 +1,7 @@
+let firstWords =["easy","complicated","strong","boring","hard","interesting","good","bad","bold","italics","regular"];
+let secondWords =["red","blue","green","yellow","black","white","violet","indigo","airplane","telephone",];
```

Comment:
Sweet; We could also use `const` here to declare these variables that hold these collections.